### PR TITLE
Update Features Section

### LINKED ISSUE #81: [FEATURE] Make features section interactive
### What?

Make features section in the Landing page interactive and more attractive. 
<p align="center">
image
</p>

### Why?

It's very static and it's calling zero the attention of the users. 

### How?

Open to ideas. Please, feel free to discuss, share them in this issue and come up with ideas to improve this.

### PR DESCRIPTION
**Pull request by ppranay20**
Fixes #81 

Improved Features Section

Here is the video.

https://github.com/user-attachments/assets/b6975b19-b036-4516-9a8f-7427d75459e4

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -4,8 +4,43 @@ import Link from "next/link";
 import { buttonVariants } from "./components/ui/button";
 import { cn } from "@/lib/utils";
 import { motion } from 'framer-motion'
+import { BarChart3, Brain, Cloud, Upload, UserCircle2, Shield } from 'lucide-react';
+import { useState } from "react";
```
Comment (manuelalferez) [inline src/app/page.tsx:8]:
```suggestion
```

Comment (manuelalferez) [reply src/app/page.tsx:8]: It's not being used

**Review by manuelalferez — APPROVED** [review]
Very clean code! Great job, as usual. Thanks, it was easy to review

**Comment by ppranay20** [comment]
Thanks @manuelalferez

**Pull request merged**